src/operation/operation.js

```diff
@@ -124,7 +124,9 @@ export default class Operation {
                     required: true,
                     content: {
                         [this.consumes || DEFAULT_PRODUCES]: {
-                            schema: this.input.toRef().build(openapiVersion),
+                            schema: {
```

Comment:
The `Reference` object is now officially the *value type* for a `$ref` key. 

That is: `{ $ref: someRef }` gets replaced with `{ $ref: someRef.build() }`